app/views/LicensorViewComponent.vue

```diff
@@ -284,6 +348,38 @@ module.exports = Vue.extend({
             forms.setErrorToProperty(el, 'createClient', 'Something went wrong')
             return
 
+    onEditApiClient: co.wrap ->
+        el = $('#client-form')
+        requiredProps = ['clientName', 'minimumLicenseDays', 'manageLicensesViaUI', 'manageLicensesViaAPI', 'revokeLicensesViaUI', 'revokeLicensesViaAPI', 'manageSubscriptionViaAPI', 'revokeSubscriptionViaAPI']
+        data = @runValidation(el, requiredProps)
+        unless data
+            return
+
+        try
+            apiClient = yield api.apiClients.getByName(data.clientName)
+            unless apiClient.length>0
+                forms.setErrorToProperty(el, 'editClient', 'API client not found')
+                return
+            attrs = {
```

Comment:
This seems to be duplicated across create and update.  Let's consolidate any common code somewhere instead of repeating ourselves.